### PR TITLE
chore: improved the project setup process

### PR DESCRIPTION
**Pull request by pranshuchittora**
**Summary**
I recently formated my laptop, while setting up the project `webpack-cli` project I found the initial setup cycle of the codebase quite weird.

**Current behaviour**
1. Clone
2. npm i (to install the node_modules)
3. npm run bootstrap (this is quite weird, `npm run clean:all && npm install && lerna bootstrap`)

**Another approach**
What if we run `lerna` directly, but it won't work as lerna is not yet installed.

**Solution**
Install node_modules in the root so that lerna gets installed, then we can run `lerna` to install the deps recursively.

In short, we are installing node_modules and then deleting them.

**Does this PR introduce a breaking change?**
No

**What kind of change does this PR introduce?**
kinda feature/build 

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
No, I have to.

**Other information**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   ],
   "scripts": {
     "bootstrap": "npm run clean:all && npm install && lerna bootstrap",
+    "setup": "npm install && lerna bootstrap",
```
Comment (rishabh3112) [inline package.json]: `bootstrap` command does that.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -67,12 +67,12 @@ In case you are suggesting a new feature, we will match your idea with our curre
...
     ```bash
     #npm
-    npm install
+    npm run setup
     npm link
     npm link webpack-cli
     ---------------------------
     #yarn
-    yarn
+    yarn run setup
```
Comment (rishabh3112) [inline .github/CONTRIBUTING.md]: I think strictly enforcing new contributors to install dependencies for side packages isn't a good idea as PR may not be related to the sub packages and just adds extra friction.
If user wants to, they should follow the steps at "Bootstrap all the submodules before building for the first time" heading below.

**Comment by pranshuchittora** [comment]
> I don't think we need a setup step

Generally `bootstrap` is the single entry point for end to end project setup.
https://docs.firefox-dev.tools/getting-started/build.html#building-and-running-locally

What about these changes
-> Bootstrap => Install Modules for the entire project. (One-step setup)
-> clean:all => Clean the modules just like the current behaviour of `bootstrap`. (Already there)

**Comment by ematipico** [comment]
I think we are fine like this for now. Please next time open an issue and let's discuss that there, so won't spend time on a PR.
